tests/cases/standalone/common/flow/flow_null.sql

```diff
@@ -105,8 +105,8 @@ VALUES
 ADMIN FLUSH_FLOW('calc_ngx_country');
 
 SELECT
-    "ngx_access_log.client",
-    "ngx_access_log.country",
+    client,
+    country,
```

Comment:
what's this change for?

Reply:
> cc @discord9

this auto gen column name come directly from datafusion, it's their rule about select a column's name changed from "src_table.col_a" to just "col_a"